api.bs

```diff
@@ -2188,7 +2198,7 @@ and a [=list=] of [=integers=] |histogram|:
     1.  Let |pkR| be the public key of the corresponding role from
         the [=aggregation service=] [HPKE configuration](https://datatracker.ietf.org/doc/html/draft-ietf-ppm-dap-15#section-4.5.1)
         obtained for the [=aggregation service=]
-        indicated by |options|.[=validated conversion options/Aggregation Service=].
+        indicated by |options|.[=validated conversion options/Aggregation Service URL=].
```

Comment:
This means that the URL for the aggregation service points at a configuration.  I guess that's fine, but it also means that it might not be where reports are submitted.

Reply:
This just says "indicated by", meaning there could be an additional mapping maintained by the browser from this URL to some other one for the purpose of key lookup.